src/main/java/pl/karolpat/service/UserService.java

```diff
@@ -0,0 +1,88 @@
+package pl.karolpat.service;
+
+import java.util.List;
+import java.util.Map;
+
+import pl.karolpat.entity.User;
+
+/**
+ * @author karolpat
+ *
+ */
+public interface UserService {
+
+	/**
+	 * Gives list of all present User instances.
+	 * 
+	 * @return List of all present User instances.
+	 */
+	List<User> getAllUsers();
```

Comment:
i to się wszystko zmieści w pamięci za parę lat?

Reply:
Podobnie, jak wcześniej - rozumiem, że paginacja rozwiązuje ten problem?